src/Juvix/Compiler/Internal/Pretty/Base.hs

```diff
@@ -91,8 +91,17 @@ instance PrettyCode Let where
     return $ kwLet <+> letClauses' <+> kwIn <+> letExpression'
 
 instance PrettyCode LetClause where
+  ppCode :: forall r. Member (Reader Options) r => LetClause -> Sem r (Doc Ann)
   ppCode = \case
     LetFunDef f -> ppCode f
+    LetMutualBlock b -> ppMutual b
+    where
+      ppMutual :: MutualBlock -> Sem r (Doc Ann)
+      ppMutual m@(MutualBlock b)
+        | [_] <- toList b = ppCode b
+        | otherwise = do
+            b' <- ppCode m
+            return (kwMutual <+> braces (line <> indent' b' <> line))
```

Comment:
Where should / can we write the `mutual` keyword? Could you add a test that demonstrates its use?

Reply:
we cannot write the `mutual` keyword, but I thought it would be useful to print it in `Internal` syntax as a means to easily see for debugging purposes what are the actual mutual blocks